### PR TITLE
Highlight disabled text field if it is not valid

### PR DESCRIPTION
**Pull request by vadymmarkov**
@3lvis @zenangst 

Validation works for disabled fields as well, but they were not highlighted with `invalidColor`. This PR aims to fix that.

### CODE REVIEW

File: Source/Cells/Text/FORMTextFieldCell.m [hunk 1]
```diff
@@ -177,6 +177,8 @@ - (void)updateFieldWithDisabled:(BOOL)disabled {
 - (void)updateWithField:(FORMField *)field {
     [super updateWithField:field];
 
+    [self validate];
```
Comment (vadymmarkov) [inline Source/Cells/Text/FORMTextFieldCell.m]: Let me know if you guys see any implication of calling `validate` here. I's done to update `isValid` property considering the new value whenever the field is changed.

**Comment by vadymmarkov** [comment]
I see https://github.com/hyperoslo/Form/pull/549 fixes Travis, so it's better to get it merged before this one I guess.

**Comment by zenangst** [comment]
@vadymmarkov 🚀

**Comment by zenangst** [comment]
Lets try and fix the Travis build :)

**Pull request merged**